justext/__main__.py

```diff
@@ -105,7 +105,7 @@ def output_detailed(paragraphs, fp=sys.stdout):
             paragraph.cf_class,
             int(paragraph.heading),
             paragraph.xpath,
-            cgi.escape(paragraph.text)
+            html.escape(paragraph.text, quote=False)
```

Comment:
Please, keep the previous behaviour preserver and don't set `quote=False`.

Reply:
Thanks you for your quick review.

The`cgi.escape` function has a `quote=None` default parameter whereas the `html.escape` has `quote=True`. To keep the same behavior, I think it's better to explicit the value of this parameter.

Reply:
Oh, didn't notice the change. Thanks for the explanation :)